doku/models/schemas/template.py

```diff
@@ -63,20 +63,36 @@ def update(cls, commit=True):
 class StylesheetSchema(DokuSchema, DateSchemaMixin, ApiSchemaMixin):
     class Meta:
         model = Stylesheet
-        exclude = ("base_templates", "templates")
         load_instance = True
 
-    API_NAME = "template"
+    API_NAME = "stylesheet"
 
     id = auto_field()
     name = NotEmptyString()
     source = auto_field()
-    base_templates = Nested("TemplateSchema", exclude=("base_style",), many=True)
+    base_templates = Nested("TemplateSchema", exclude=("base_style","styles"), many=True)
     templates = Nested("TemplateSchema", exclude=("styles",), many=True)
 
     upload_url = fields.Method("_upload_url", dump_only=True, allow_none=True)
+    delete_url = fields.Method("_delete_url", dump_only=True, allow_none=True)
```

Comment:
The `delete_url` is already defined and does not need to be overwritten. 

https://github.com/desklab/doku/blob/92bbfecc7e5c06cbf9b6b5ec67b4af15f9eabdc3/doku/models/schemas/common.py#L55

It's enough to just overwrite `def delete(. . .)`.

Reply:
fixed in 18ca3d14a8ab0fc53f87d93a1b0fb2397506a635